### DISCUSSION TITLE
[Web] Automatic lock screen

### DISCUSSION THREAD

**Discussion by vishnukvmd** [ente-io/ente#484] [Enhancements]
If the tab remains open for over a specific duration (say 5 minutes), the app could bring up the lockscreen.

**Comment by Niranjan-Dorage** [comment]
issue still active?
@ua741 @vishnukvmd

**Reply by nathfavour** [reply]
should be...

**Comment by nathfavour** [comment]
This would be a very good idea

**Comment by nathfavour** [comment]
especially for ente auth web

**Comment by nathfavour** [comment]
> If the tab remains open for over a specific duration (say 5 minutes), the app could bring up the lockscreen.

any update? @vishnukvmd

**Comment by fadookie** [comment]
This seems important, when I walk away from my computer it would be best if I could set an auto-lock to prevent my OTPs from being accessed.
